nbextensions/usability/toc2/main.js

```diff
@@ -307,7 +333,14 @@ var table_of_contents = function () {
           ul= ul.parent();
           while(!ul.is('ul')){ ul= ul.parent(); }
       }
-
+      // Change link id -- append current num_str so as to get a kind of unique anchor 
```

Comment:
Yes, all ids are changed, but all links are updated accordingly; so this essentially works. [Detection of non unique toc links would add overhead.]